registration/api/v2/views.py

```diff
@@ -433,17 +433,22 @@ def get_queryset(self):
         For a regular user it should return a single object (for now).
         """
         user: User = self.request.user
+        queryset = BetaTestApplication.objects.select_related("instance")
+
         if not user.is_authenticated:
-            return BetaTestApplication.objects.none()
+            return queryset.none()
         elif user.is_staff:
-            return BetaTestApplication.objects.all()
+            return queryset.all()
         else:
-            return BetaTestApplication.objects.filter(user=self.request.user)
+            return queryset.filter(user=self.request.user)
 
     def get_serializer_class(self):
         if self.action == "create":
             return OpenEdXInstanceDeploymentCreateSerializer
 
+        if self.action == "list":
+            return OpenEdXInstanceDeploymentAlertSerializer
+
```

Comment:
Rather than this being the list action for deployments, I think we should create a completely new View. The reasoning here is:
- Eventually we might need a real deployment list, and this will be the natural place for it. 
- Eventually we will have more kinds of alerts, notifications etc than just those about instances/deployments.